### PR TITLE
Retrieve Config from GitHub

### PR DESCRIPTION
**Pull request by danieledamiani**

### CODE REVIEW

File: packages/tempus-core-services/src/constants.ts [hunk 1]
```diff
@@ -54,10 +54,9 @@ export const tokenPrecision: { [ticker in Ticker]?: number } = {
   YFI: 18,
 };
 
-const ALCHEMY_KEY = process.env.REACT_APP_MAINNET_ALCHEMY_KEY;
-const GOERLI_ALCHEMY_KEY = process.env.REACT_APP_GOERLI_ALCHEMY_KEY;
-const FANTOM_ENDPOINT = process.env.REACT_APP_FANTOM_ENDPOINT;
-const REACT_APP_ETHEREUM_FORK_RPC = process.env.REACT_APP_ETHEREUM_FORK_RPC;
+const { ALCHEMY_KEY, GOERLI_ALCHEMY_KEY, FANTOM_ENDPOINT, REACT_APP_ETHEREUM_FORK_RPC } = process.env;
+
+export const GITHUB_TOKEN = process.env.REACT_APP_GITHUB_TOKEN;
```
Comment (lazarmitic) [inline packages/tempus-core-services/src/constants.ts:59]: This token will be bundled and visible in users browsers. What kind of permissions this token grants user?

Comment (danieledamiani) [reply packages/tempus-core-services/src/constants.ts:59]: At the moment these ones
Screenshot 2022-04-27 at 15 49 11

Comment (lazarmitic) [reply packages/tempus-core-services/src/constants.ts:59]: Repo access grants full control of private repositories. Can we make config repo public and skip token usage?

Comment (danieledamiani) [reply packages/tempus-core-services/src/constants.ts:59]: I guess yes

**Review by lazarmitic — APPROVED** [review]

**Pull request merged**